app/actions/cpacpy.js

```diff
@@ -168,13 +168,14 @@ export function scheduleDataSettings(scheduler, dataSettings, { success, error }
   )
 }
 
-export function schedulePipeline(scheduler, dataConfig, pipeline, { success, error }){
+export function schedulePipeline(scheduler, dataConfig, pipeline, { success, error }, profile= {}){
```

Comment:
Just for conformity, I would suggest moving the callbacks to the last position, leaving 'profile' after 'pipeline'. Also, the profile should not be optional (so no `= {}`).